extension/src/parameterDefinitions.ts

```diff
@@ -16,7 +16,7 @@ const REMOTELOG_LUA_LOG_LEVELS: string[] = ["NONE", "FATAL", "ERROR", "WARN", "I
 const LOG_LEVEL_OPTIONS: SelectOption[] = REMOTELOG_LUA_LOG_LEVELS.map(level => { return { id: level, name: level } })
 
 const allParams: AllParameters = {
-    virtualSchemaName: { id: "virtualSchemaName", name: "Name of the new virtual schema", type: "string", required: true },
+    virtualSchemaName: { id: "virtualSchemaName", name: "Virtual Schema name", type: "string", required: true, regex: "[a-zA-Z_]+", description: "Name for the new virtual schema", placeholder: "MY_VIRTUAL_SCHEMA" },
```

Comment:
```suggestion
    virtualSchemaName: { id: "virtualSchemaName", name: "Virtual Schema name", type: "string", required: true, regex: "[a-zA-Z][a-zA-Z0-9_]*", description: "Name for the new virtual schema", placeholder: "MY_VIRTUAL_SCHEMA" },
```

Reply:
This will be fixed in https://github.com/exasol/extension-manager-interface/issues/56